### ISSUE TITLE
Create time_bnds for output netCDF files

### ISSUE THREAD

**Issue by huard** [Ouranosinc/xclim#74]
### Description
Compute time_bnds and attach them to output. 

Can probably be done using OCGIS, also see https://github.com/pydata/xarray/issues/2481 for xarray functionality that could help go into this direction.

**Comment by tlogan2000** [comment]
This could be potentially solved by resampling the ds.time with the same frequency as the indicator and exratcing min / max

**Comment by tlogan2000** [comment]
@huard here is the bit of code that I was talking about (I used it when making mf-datasets filenames but it could maybe serve for the time bounds as well)
```
freq = 'YS'
timestamps = ds.time.resample(time=freq)

for n, i in enumerate(zip(timestamps.min(),timestamps.max())):
    print('year', n, 'start', i[0], '\nyear',n,'end', i[1], '\n\n')

**Comment by Zeitsperre** [comment]
With the merge of #164 I'm pushing this up to v0.15. Work for you, @huard?

**Comment by huard** [comment]
Sure. Want to include the PR for the doc refactoring in it or not ?

**Comment by Zeitsperre** [comment]
If it's feasible, why not? I need to speak with @aulemahal to see if there's anything that I can help with with regards to that PR.

**Comment by aulemahal** [comment]
@huard  Is this a priority? It could be done for 0.15 or we wait for the CRIM's suggestion for the refactoring and *then* we implement this?

**Comment by huard** [comment]
@aulemahal No, this is a bonus feature.

**Comment by huard** [comment]
We can't add a coordinate with a new dimension (nbnd) to a DataArray. This would only work on a dataset. What do we do with this ?

**Comment by huard** [comment]
See https://github.com/andersy005/xtimeutil

**Comment by Zeitsperre** [comment]
Closed with #1207
